<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(247, 252, 255);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="LoanRegistration" name="loanRegistration" native="true"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 200, 200);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 166, 166);</string>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Borrowers"/>
    <addaction name="actionShow_Librarians"/>
    <addaction name="actionBook_list"/>
    <addaction name="actionLoan_register"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockBorrowersWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Borrowers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="BorrowerRegister" name="borrowerRegister"/>
  </widget>
  <widget class="QDockWidget" name="dockBooksWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Books</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="booksRegister"/>
  </widget>
  <widget class="QDockWidget" name="dockLibrarianWidget">
   <property name="windowTitle">
    <string>Librarians</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="LibrarianRegister" name="librarianRegister"/>
  </widget>
  <widget class="QDockWidget" name="dockLoanWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Loans</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="LoanRegister" name="loanRegister"/>
  </widget>
  <action name="actionShow_Borrowers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Borrowers</string>
   </property>
  </action>
  <action name="actionShow_Librarians">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Librarians</string>
   </property>
  </action>
  <action name="actionBook_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Book list</string>
   </property>
  </action>
  <action name="actionLoan_register">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loan register</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/exit.ico</normaloff>:/resource/img/exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/img/about.ico</normaloff>:/resource/img/about.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BorrowerRegister</class>
   <extends>QWidget</extends>
   <header>registerwidget/borrowerregister.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibrarianRegister</class>
   <extends>QWidget</extends>
   <header>registerwidget/librarianregister.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoanRegister</class>
   <extends>QWidget</extends>
   <header>registerwidget/loanregister.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoanRegistration</class>
   <extends>QWidget</extends>
   <header>loanregistration.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
